roles/generate-jenkins/templates/DOCUMENTATION.j2

```diff
@@ -1,7 +1,16 @@
 ---
 title: {{ project_name }}
 ---
+{% if project_deprecation_status %}
+# DEPRECATION NOTICE
+
+This image is deprecated. It will no longer receive automated updates.
+{% if project_deprecation_message is defined %}{{ project_deprecation_message }}{% endif %}
+
+{% endif %}
 # [{{ lsio_project_name_short }}/{{ project_name }}]({{ lsio_github_url }}/{{ project_repo_name }})
+<!-- DO NOT EDIT THIS FILE MANUALLY  -->
+<!-- Please read the {{ lsio_github_asset }}/.github/CONTRIBUTING.md -->
```

Comment:
Do we want this here? adding it under the metadata makes sense in my eyes

Reply:
Resolved in https://github.com/linuxserver/docker-jenkins-builder/pull/101/commits/c5550f163cc6d2814c5cad2e1b895a1d7d8226e3